### PR TITLE
feat(website/header): add js links 

### PR DESCRIPTION
**Pull request by mohamedMok**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [x] No

## Describe the changes

GitHub issue number or Jira issue URL: N/A

### CODE REVIEW

**Comment by nherwegh** [comment]
@mohamedMok can we have new tabs on click on links?

File: packages/gatsby-theme-styleguide/src/gatsby-components/JSImplementation/JSImplementation.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  line-height: 1.5;
+  text-transform: capitalize;
+`
+
+const StatusName = styled.span`
+  text-decoration: underline;
+`
+
+const StatusContainer = styled.div`
+  display: inline-block;
```
Comment (tiloyi) [inline packages/gatsby-theme-styleguide/src/gatsby-components/JSImplementation/JSImplementation.js:26]: I think it's not necessary to have a `display: inline-block` here.

File: packages/gatsby-theme-styleguide/src/gatsby-components/JSImplementation/JSImplementation.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    .map(([key, value], index) => {
+      if (!key || !value) {
+        return false
+      }
+      return (
+        <StatusItem key={index}>
+          {value.link !== '' ? (
+            <StatusText>
+              <StatusName>{key}</StatusName>
+              <Link target="_blank" href={value.link}>
```
Comment (tiloyi) [inline packages/gatsby-theme-styleguide/src/gatsby-components/JSImplementation/JSImplementation.js:50]: I think you can't use the `target="_blank"` attribute with the `<Link>` component of Gatsby. If you want to open links in `target="_blank"`, it seems to me that you have to use a `<a>` tag.

=>https://www.gatsbyjs.com/docs/linking-between-pages/

File: packages/gatsby-theme-styleguide/src/gatsby-components/PatternStatusGroup/PatternStatusGroup.js [hunk 3]
```diff
@@ -13,8 +13,14 @@ const StatusItem = styled.span`
...
-  font-weight: bold;
-  margin-right: 0.5rem;
+  font-weight: normal;
+  font-size: 1rem;
+  line-height: 1.5;
+  text-transform: capitalize;
+`
+
+const StatusContainer = styled.div`
+  display: inline-block;
```
Comment (tiloyi) [inline packages/gatsby-theme-styleguide/src/gatsby-components/PatternStatusGroup/PatternStatusGroup.js:23]: I think it's not necessary to have a `display: inline-block` here.

**Review by tiloyi — APPROVED** [review]

**Pull request merged**